src/manifest.json

```diff
@@ -9,7 +9,7 @@
         "description": "Track and limit time your activity in the browser every day.",
 
         "options_page": "options.html",
-    
+
```

Comment:
same

Reply:
I didn't add any empty lines intentionally in manifest.json. All these lines are already present in master-branch.